pep_sphinx_extensions/pep_processor/transforms/pep_headers.py

```diff
@@ -115,3 +117,24 @@ def _pretty_thread(text: nodes.Text) -> nodes.Text:
     except ValueError:
         # archives and pipermail not in list, e.g. PEP 245
         return text
+
+
+def _process_post_history(body: nodes.field_body) -> list[nodes.Text | nodes.reference]:
+    new_nodes = []
+    for pair in body.astext().split(","):
+        pair = pair.strip()
+        try:
+            # if Post-History has no links, ``pair.split(maxsplit=1)``
+            # will raise ValueError
+            date, uri = pair.split(maxsplit=1)
+            node = nodes.reference("",
+               date.strip(),
+               refuri=uri.strip(" \f\n\r\t><"),
+               internal=False
+           )
+        except ValueError:
```

Comment:
Should we instead fail if the field doesn't parse properly? It would be good to give that feedback to authors.

Reply:
True, I guess we need some way to avoid erroring on anything that's already there, while alerting on things that look like the desired syntax but aren't quite the same.

Also, wouldn't this get a false positive on PEP 3135:
```
pep-3135.txt:Post-History: 28-Apr-2007, 29-Apr-2007 (1), 29-Apr-2007 (2), 14-May-2007, 12-Mar-2009
```

Reply:
Also PEP 551:
```
pep-0551.rst:Post-History: 24-Aug-2017 (security-sig), 28-Aug-2017 (python-dev)
```

Reply:
Perhaps a normalisation pass is needed here. (Or ignoring anything that doesn't start with "http")

A

Reply:
I wouldn't mind us changing the handful of PEPs with very eccentric Post-History formats (648, 538, 467, 367, 311 also stand out; and 424 and 428 have bare URLs).